packages/visual-editor/src/components/Draggable/DraggableComponent.tsx

```diff
@@ -38,7 +38,7 @@ export const DraggableComponent = ({
   onMouseUp?: (e: SyntheticEvent) => void;
   onMouseOver?: (e: SyntheticEvent) => void;
   onMouseOut?: (e: SyntheticEvent) => void;
-  coordinates: Rect;
+  coordinates: Rect | null;
```

Comment:
I noticed there's a couple more components like Tooltip and DraggableChildComponent that define this coordinates prop as `Rect` instead of `Rect | null` - if you change those instances it will prevent the same build warnings in those components as well.